tests/unit/versioning/VersionID.js

```diff
@@ -39,4 +39,48 @@ describe('test generating versionIds', () => {
         assert.strictEqual(vids.length, count);
         assert.deepStrictEqual(vids, decoded);
     });
+
+    it('simple tiny version test', () => {
```

Comment:
Do you have other test cases available? E.g. with version IDs of different length, I believe they can vary by one to a few characters in length.